src/pages/pt-BR/core-concepts/partial-hydration.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Hidratação Parcial no Astro
+description: Aprenda como a hidratação parcial funciona usando a "Arquitetura em Ilhas" no Astro.
+i18nReady: true
+---
+
+**Astro gera todo website com zero JavaScript no lado do cliente, por padrão.** Use qualquer componente de UI frontend que você queira ([React](https://reactjs.org/), [Preact](https://preactjs.com/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/), [SolidJS](https://www.solidjs.com/), [AlpineJS](https://alpinejs.dev/) e [Lit](https://lit.dev/)) e o Astro irá automaticamente renderizá-lo como HTML em tempo de build e remover todo o JavaScript. Isso faz todos os sites rápidos por padrão.
+
+
+```astro
+---
+// Exemplo: Use um componente React estático na página, sem JavaScript.
+import MyReactComponent from '../components/MyReactComponent.jsx';
+---
+<!-- 100% HTML, Zero JavaScript! -->
+<MyReactComponent />
+```
+
+Porém as vezes, JavaScript no lado do cliente é necessário para criar interfaces de usuário interativas. Quando você se ver precisando de UI interativa na página, Astro não te força a utilizar JavaScript em 100% da página. Em vez disso, Astro utiliza uma técnica chamada **hidratação parcial** que te deixa hidratar componentes individualmente na página. Dessa forma, você envia apenas o JavaScript absolutamente necessário que você precisa para carregar a página.
+
+
+```astro
+---
+// Exemplo: Use um componente React dinâmico na página.
+import MyReactComponent from '../components/MyReactComponent.jsx';
+---
+<!-- Este componente agora é interativo na página! 
+     O resto do seu website continua o mesmo. -->
+<MyReactComponent client:load />
+```
+
+A maioria do seu site continua como puro e leve HTML e CSS, com isoladas **ilhas de interatividade**.
+
+## Hidratação Parcial
+
+Há vários casos onde você precisa de um componente de UI interativo que rode no navegador:
+
+- Um carrossel de imagens
+- Uma barra de pesquisa com autocompletação
```

Comment:
What do you think about autocomplete being translated to `preenchimento automático`?

Reply:
I like it! It was the exact word I was looking for, thanks!